CHANGES.rst

```diff
@@ -3,6 +3,7 @@
 
 - Remove duplicate unit manifests [#5]
 - Increase setuptools version [#6]
+- Fix branch name in ci [#7]
```

Comment:
```suggestion
```

CI changes do not need changelogs

Reply:
This is what the `no-changelog-entry-needed` label is for.

Reply:
What is the threshold for a change that should be added to the changelog?